### PR TITLE
Jesse web blog

### PR DESCRIPTION
**Pull request by JesseWeb**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.
e.g.
```
---
 
 title: "'Score Your Face Photo' a ML&Wechaty practice"
 date: 2017-09-18 09:00 +0800
 author: huyingxi
 ---
 
 > Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```
- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to the blog `/download/2018` directory.
- [x] Add `<!--more-->` section to just show the abstract in blog homepage.
- [x] Add teaser image for the blog
- [x] Add your self as an author to the author data file at https://github.com/Chatie/blog/blob/master/docs/_data/authors.yml with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

[See more](https://github.com/Chatie/blog#contribute-guideline)

### CODE REVIEW

**Comment by JesseWeb** [comment]
@huan what should I do for this check fail?

**Comment by JesseWeb** [comment]
ci fail detail:
```
# image size should not more than 1MB
58
ok 2 should get image file list
59
not ok 3 ../assets/2020/promote-bot/banner.png exceed the max limit: width: 2208, size: 71267.
```

image Size:

image

as you can see, the image size is not out of max limit

**Comment by su-chang** [comment]
> ../assets/2020/promote-bot/banner.png exceed the max limit: width: 2208, size: 71267. use "./scripts/fit-image.sh <FILE>" to adjust it fit.

Try adjust fit by script?

**Comment by JesseWeb** [comment]
> > ../assets/2020/promote-bot/banner.png exceed the max limit: width: 2208, size: 71267. use "./scripts/fit-image.sh " to adjust it fit.
> 
> Try adjust fit by script?

where can I find this script in project?

**Comment by JesseWeb** [comment]
> > > ../assets/2020/promote-bot/banner.png exceed the max limit: width: 2208, size: 71267. use "./scripts/fit-image.sh " to adjust it fit.
> > 
> > 
> > Try adjust fit by script?
> 
> where can I find this script in project?

never mind. I found it.

**Comment by lijiarui** [comment]
@JesseWeb Thanks for this PR， please remove the unrelated files then I can merge this PR.

**Review by lijiarui — CHANGES_REQUESTED** [review]
too many unrelated files, it should be clean.

**Comment by JesseWeb** [comment]
I’ll solve this tomorrow

**Comment by JesseWeb** [comment]
@lijiarui those unrelated files are cleaned.

File: _posts/2020-04-30-wechat-promote-tool.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+![INVITE](/assets/2020/promote-bot/result1.png)
+
+## 后续计划
+
+* 自动判断群所属用户、无需手动绑定。
+* 客服功能
+* 增加更多推广能力
+
+## 鸣谢
+感谢 [Wechaty](https://github.com/wechaty/wechaty) 开源项目及 JuziBot 公司提供的 token，为开发者带来极大便利！
```
Comment (lijiarui) [inline _posts/2020-04-30-wechat-promote-tool.md]: You can add link to JuziBot to， something like this:
`[句子互动](https://www.juzibot.com/)`

Thanks!

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: _posts/2020-04-30-wechat-promote-tool.md (hunk 1 above)
Comment (JesseWeb) [reply _posts/2020-04-30-wechat-promote-tool.md]: Yea no problem.

**Comment by JesseWeb** [comment]
change request has done.

**Comment by lijiarui** [comment]
Good job！

**Review by lijiarui — APPROVED** [review]

**Comment by lijiarui** [comment]
Congratulations! :tada: :tada: :tada:

You become a wechaty contributor successfully!  :clap: :clap: :clap:

Please: 
- [ ] add my wechat: `ruirui_0914` and I will invite you into Wechaty Contributor Room
- [ ] join wechaty org by clicking https://github.com/wechaty

**Comment by JesseWeb** [comment]
my pleasure

**Pull request merged**